pom.xml

```diff
@@ -55,7 +55,7 @@
         <!-- can't upgrade jersey past 2.25 until we move to java 8 -->
         <jersey.version>2.25</jersey.version>
```

Comment:
maybe we can also update jersey to 2.31

Reply:
noticed here: https://confluentinc.atlassian.net/browse/CPKAFKA-6127. Will add that change.